www/FingerPrint.js

```diff
@@ -57,9 +57,23 @@ FingerPrint.prototype.authenticate = function (successCallback, errorCallback,ob
 			errorCallback(error);
     };
     // if the device is android call th FingerprintAuth plugin
-    if(cordova.platformId === "android"){FingerprintAuth.show(object, successCbFingerPrintAuth, errorCbFingerPrintAuth);}
+    if(cordova.platformId === "android"){FingerprintAuth.encrypt(object, successCbFingerPrintAuth, errorCbFingerPrintAuth);}
     // if the device is ios call the touchid plugin 
     if(cordova.platformId === "ios")touchid.authenticate(successCallback, errorCallback, object.dialogMessage);
 };
 
+FingerPrint.prototype.checkBiometry = function(successCallback, errorCallback) {
+    // success callback function for android 
+    var successCbFingerPrintAuth = function(result) {successCallback(1)};
+    // error callback function for android 
+    var errorCbFingerPrintAuth = function(error) {
+			errorCallback(-1);
```

Comment:
Don't want to nitpick but this is missing style consistency. on line 67 the function is a one-liner, in line 69 it's a multi-line function and the body is not formatted.